server/controller/apis/topicController.js

```diff
@@ -259,15 +228,18 @@ exports.saveTopic = async ( req, res, redirect ) => {
 
     if(authUserId > 0) {
 
-        topic.id = req.body.topicId;
+        topic.id = req.body.id;
```

Comment:
This might be to work with the current UI, but are we moving to a model where we are using topicId (entitynameId?)